yaml/screenconnect.yaml

```diff
@@ -56,7 +56,17 @@ Artifacts:
   - File: C:\ProgramData\ScreenConnect Client*\user.config
     Description: ScreenConnect client user configuration
     OS: Windows
-  EventLog: []
+  EventLog:
+  - EventID: 7045
+    ProviderName: ["ScreenConnect", "ScreenConnect Client (<hex string>)"]
+    LogFile: Application.evtx
+    ServiceName: ScreenConnect Client (<hex string>)
+    Description: Service installation event as a result of ScreenConnect installation.
+  - EventID: 20
+    ProviderName: ["ScreenConnect", "ScreenConnect Client (<hex string>)"]
```

Comment:
According to https://community.spiceworks.com/t/screenconnect-client-side-logging/954078
Provider name should be "ScreenConnect Client ()"

Reply:
Removed <hex string> and standardized provider as "ScreenConnect Client ()"